rs3_trading/utils/utilities.py

```diff
@@ -14,10 +17,50 @@ def pop_unix_time_from_dict(data: dict) -> int:
     return unix_time
 
 
-def transform_to_tick_dataframe(data: dict, unix_time: int) -> pd.DataFrame:
-    column_names = ['item_name', 'price']
+def transform_to_dataframe(data: dict, unix_time: int, column_name: str) -> pd.DataFrame:
+    column_names = ['item_name', column_name]
     ge_tick_dataframe = pd.DataFrame.from_dict(data, orient='index').reset_index()
     ge_tick_dataframe.columns = column_names
     ge_tick_dataframe['time'] = pd.to_datetime(unix_time, unit='s')
     ge_tick_dataframe['item'] = ge_tick_dataframe['item_name'].str.lower().replace('[^A-Za-z0-9+()]+', '', regex=True)
     return ge_tick_dataframe
+
+
+def create_database():
+    con = duckdb.connect('GE_Tick_Data')
```

Comment:
You can replace all of the

```
con = duckdb.connect('db')

# CODE HERE

con.close()

```

With a context manager. I would google duckdb context manager